handlers/main.yml

```diff
@@ -11,3 +11,7 @@
   sudo: yes
   command: timedatectl set-ntp true
   notify: Reload systemd-timesyncd
+
+- name: Run logrotate
+  become: yes
+  command: /usr/sbin/logrotate -fv /etc/logrotate.d/docker-container-logrotate
```

Comment:
Yep, that's what we are doing, just copying the config. We don't need to reload the service, it'll pick up the config based on its timer. Restarting the service doesn't alter the timer and the cron would still be scheduled at the same time.

Which is why we added this handler to explicitly call the logroate daemon to validate/run the logrotate once.